lib/kits/core-ui/components/helpers/component.js

```diff
@@ -0,0 +1,58 @@
+const componentHelper = {
```

Comment:
Please `export`(and `import`) each of these helpers separately, having a big object like this is not conducive to dead code elimination(and eventually splitting up the SDK)